portafly/src/App.tsx

```diff
@@ -17,60 +19,52 @@ const navItems = [
     title: 'Analytics',
     to: '/analytics',
     items: [
-      {to: '/analytics/usage', title: 'Usage'},
+      { to: '/analytics/usage', title: 'Usage' },
     ],
   },
   {
     title: 'Applications',
     to: '/applications',
     items: [
-      {to: '/applications', title: 'Listing'},
-      {to: '/applications/plans', title: 'Application Plans'}
+      { to: '/applications', title: 'Listing' },
+      { to: '/applications/plans', title: 'Application Plans' },
     ],
   },
   {
     title: 'Integration',
     to: '/integration',
     items: [
-      {to: '/integration/configuration', title: 'Configuration'}
-    ]
-  }
+      { to: '/integration/configuration', title: 'Configuration' },
+    ],
+  },
 ]
 
 const getOverviewPage = () => import('./pages/Overview')
 const getApplicationsPage = () => import('./pages/Applications')
 
-const App = () => {
-  return (
-      <Router>
-        <LastLocationProvider>
-          <Root />
-        </LastLocationProvider>
-      </Router>
-  )
-}
+const App = () => (
+  <Router>
+    <LastLocationProvider>
+      <Root />
+    </LastLocationProvider>
+  </Router>
+)
 
-const PagesSwitch = () => {
-  return (
-    <SwitchWith404>
-      <LazyRoute path="/" exact getComponent={getOverviewPage} />
-      <LazyRoute path="/applications" exact getComponent={getApplicationsPage} />
-      <Redirect
-        path="/overview"
-        to="/"
-        exact
-      />
-    </SwitchWith404>
-  )
-}
+const PagesSwitch = () => (
+  <SwitchWith404>
+    <LazyRoute path="/" exact getComponent={getOverviewPage} />
+    <LazyRoute path="/applications" exact getComponent={getApplicationsPage} />
+    <Redirect path="/overview" to="/" exact />
```

Comment:
what's the rule for props here? or is it the standard 80 char long to line break?

Reply:
it's 100 chars I think.
1-3 props per line (below 100 chars or 80 whatever), otherwise break and 1 per line